recipes/cxxopts/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.10)
```

```suggestion
cmake_minimum_required(VERSION 3.1.3)
```
There is no special case for Cmake 3.10.

---

Agree, no logic here is tied to CMake 3.10. Also, any CMake >= 3.1.2 is enough to use [targets approach](https://docs.conan.io/en/latest/integrations/build_system/cmake/cmake_generator.html#targets-approach).